src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIBuildJobManagementService.java

```diff
@@ -49,19 +49,26 @@ public class LocalCIBuildJobManagementService {
 
     private final LocalCIContainerService localCIContainerService;
 
+    private final LocalCIDockerService localCIDockerService;
+
     @Value("${artemis.continuous-integration.timeout-seconds:120}")
     private int timeoutSeconds;
 
     @Value("${artemis.continuous-integration.asynchronous:true}")
     private boolean runBuildJobsAsynchronously;
 
+    @Value("${artemis.continuous-integration.build.images.java.default}")
+    String dockerImage;
```

Comment:
You should probably make this private as well.
```suggestion
    private String dockerImage;
```